.github/workflows/tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       max-parallel: 6
       matrix:
         operatingSystem: [ubuntu-latest, windows-latest]
-        phpVersion: ['7.2', '7.3', '7.4', '8.0']
+        phpVersion: ['7.3', '7.4', '8.0']
```

Comment:
might as well just get rid of 7.3 support altogether if Laravel 9 isn't going to support it

Reply:
Done.